warn/warn_naming.go

```diff
@@ -65,3 +67,65 @@ func confusingNameWarning(f *build.File, fix bool) []*Finding {
 
 	return findings
 }
+
+// isProviderNode returns whether the node is a call of `provider()`
+func isProviderNode(expr build.Expr) bool {
+	call, ok := expr.(*build.CallExpr)
+	if !ok {
+		return false
+	}
+	name, ok := call.X.(*build.Ident)
+	return ok && name.Name == "provider"
+}
```

Comment:
I think we can skip this check.
We can simplify to: if it's UpperCamelCase, it should end with the suffix "Info".

Reply:
Removed.